sites/partners/__tests__/pages/application/index.test.tsx

```diff
@@ -0,0 +1,614 @@
+import ApplicationsList from "../../../src/pages/application/[id]"
+import {
+  AlternateContactRelationship,
+  LanguagesEnum,
+  UnitTypeEnum,
+  YesNoEnum,
+} from "@bloom-housing/shared-helpers/src/types/backend-swagger"
+import { AuthContext } from "@bloom-housing/shared-helpers"
+import { ApplicationContext } from "../../../src/components/applications/ApplicationContext"
+import DetailsApplicationData from "../../../src/components/applications/PaperApplicationDetails/sections/DetailsApplicationData"
+import DetailsPrimaryApplicant from "../../../src/components/applications/PaperApplicationDetails/sections/DetailsPrimaryApplicant"
+import DetailsAlternateContact from "../../../src/components/applications/PaperApplicationDetails/sections/DetailsAlternateContact"
+import DetailsHouseholdDetails from "../../../src/components/applications/PaperApplicationDetails/sections/DetailsHouseholdDetails"
+import DetailsHouseholdIncome from "../../../src/components/applications/PaperApplicationDetails/sections/DetailsHouseholdIncome"
+import DetailsTerms from "../../../src/components/applications/PaperApplicationDetails/sections/DetailsTerms"
+import DetailsHouseholdMembers from "../../../src/components/applications/PaperApplicationDetails/sections/DetailsHouseholdMembers"
+
+const server = setupServer()
+
+beforeAll(() => {
+  server.listen()
+})
+
+beforeEach(() => {
+  document.cookie = "access-token-available=True"
+  server.use(
+    rest.get("http://localhost/api/adapter/listings/Uvbk5qurpB2WI9V6WnNdH", (_req, res, ctx) => {
+      return res(ctx.json(listing))
+    }),
+    rest.get("http://localhost/api/adapter/user", (_req, res, ctx) => {
+      return res(ctx.json(user))
+    }),
+    rest.delete("http://localhost/api/adapter/applications", (_req, res, ctx) => {
+      return res(ctx.json({}))
+    })
+  )
+})
+
+afterEach(() => {
+  server.resetHandlers()
+  window.sessionStorage.clear()
+})
+
+afterAll(() => {
+  server.close()
+})
+
+describe("partners_application_index", () => {
+  it("should return null on no application", () => {
+    mockNextRouter({ id: "application_1" })
+    document.cookie = "access-token-available=True"
+    server.use(
+      rest.get("http://localhost:3100/applications/application_1", (_req, res, ctx) => {
+        return res(ctx.json({}))
+      })
+    )
+
+    const { container } = render(<ApplicationsList />)
+    expect(container.firstChild).toBeNull()
+  })
+
+  it("should display proper submission status", async () => {
+    mockNextRouter({ id: "application_1" })
+
+    document.cookie = "access-token-available=True"
+    server.use(
+      rest.get("http://localhost:3100/applications/application_1", (_req, res, ctx) => {
+        return res(ctx.json(application))
+      })
+    )
+
+    const { findByText } = render(<ApplicationsList />)
+    const submissionStatus = await findByText("Submitted")
+    expect(submissionStatus).toBeInTheDocument()
+  })
```

Comment:
question: Should these first tow tests (The ones testing `<ApplicationsList>`) be in the other index.test.tsx you touched in this PR?

Reply:
Ignore that. The actual thing that should change is the component doesn't have an appropriate name. The component in `bloom/sites/partners/src/pages/application/[id]/index.tsx` should actually be called something like `Application` instead of `ApplicationsList`. But you don't need to change it for this PR